pkg/scheduler/framework/session.go

```diff
@@ -58,7 +58,7 @@ type Session struct {
 func openSession(cache cache.Cache) *Session {
 	ssn := &Session{
 		UID:        uuid.NewUUID(),
-		cache:      cache,
+		Cache:      cache,
```

Comment:
cache here is an interface, I think I either need to cast cache interface to concrete struct to get the object, or expose methods like GetKubeClient() and GetRecorder() in Cache interface. I am very new to go, which one is better?